### PR TITLE
Cast appropriate values to Python types in Parse() functions.

### LINKED ISSUE #123: Parsers don't cast values to Python types
`ais.tag_block.Parse()` and `ais.vdm.Parse()` return dictionaries with some keys that should be cast to `int` and `float`.  For example, result from `ais.vdm.Parse()`:

```python
{
    "body": "15SimR001Q6jmk9pwgp<Rr1n06sd",
    "chan": "A",
    "checksum": "66",
    "fill_bits": "0",
    "sen_num": "1",
    "sen_tot": "1",
    "seq_id": null,
    "talker": "AI",
    "vdm": "!AIVDM,1,1,,A,15SimR001Q6jmk9pwgp<Rr1n06sd,0*66",
    "vdm_type": "VDM"
}
```

All numeric values except `checksum` should be converted to the appropriate Python type with [`ais.util.MaybeToNumber()`](https://github.com/schwehr/libais/blob/master/ais/util.py#L4-L19).  Additionally, `MaybeToNumber()` should probably add `NoneType` to the `isinsance()` checks at the top.

@schwehr I can take care of this.

### PR DESCRIPTION
**Pull request by geowurster**
Closes #123 

@schwehr I took care of `vdm`, `tag_block`, and `uscg` `Parse()` functions.  Each file defines a global `NUMERIC_FIELDS` tuple stating which fields should be converted, so let me know if I omitted or included fields that should remain a string.  I didn't see an example value for USCG `rffi` so I'll defer to you on that one.

### CODE REVIEW

File: ais/tag_block.py [hunk 1]
```diff
@@ -10,16 +10,21 @@
 TODO(schwehr): Catch a wider variety of incomplete groups.
 TODO(schwehr): Compute running stats in the queue.
 """
+
+
```
Comment (schwehr) [inline ais/tag_block.py]: Only one blank line between the doc string and the imports

File: ais/tag_block.py [hunk 2]
```diff
@@ -10,16 +10,21 @@
...
 import ais
 from ais import nmea
 from ais import nmea_messages
 from ais import util
 from ais import vdm
-import six.moves.queue as Queue
+
 
 # Added a decimal value to time beyond the normal TAG BLOCK spec.
```
Comment (schwehr) [inline ais/tag_block.py]: Again, only one blank line

File: ais/tag_block.py [hunk 3]
```diff
@@ -41,6 +46,19 @@
...
+  'group_id',
+  'line_num',
+  'rel_time',
+  'sentence_num',
+  'sentence_tot',
+  'time'
+)
+
+
+
```
Comment (schwehr) [inline ais/tag_block.py]: 2 blank lines please

File: ais/tag_block.py [hunk 4]
```diff
@@ -41,6 +46,19 @@
 """, re.VERBOSE)
```
Comment (schwehr) [inline ais/tag_block.py]: only 1 blank line here

File: ais/uscg.py [hunk 5]
```diff
@@ -37,14 +37,17 @@
 
 """
 
+
```
Comment (schwehr) [inline ais/uscg.py]: 1 blank line

File: ais/uscg.py [hunk 6]
```diff
@@ -37,14 +37,17 @@
...
 import logging
 import re
 
+import six
+import six.moves.queue as Queue
+
 import ais
 from ais import util
 from ais import vdm
-import six.moves.queue as Queue
```
Comment (schwehr) [inline ais/uscg.py]: I should have had only 1 blank line.  Can you fix my mistake?

File: ais/uscg.py [hunk 7]
```diff
@@ -66,6 +69,19 @@
 """, re.VERBOSE)
```
Comment (schwehr) [inline ais/uscg.py]: 1 blank line

File: ais/vdm.py [hunk 8]
```diff
@@ -58,14 +58,18 @@
   http://www.itu.int/rec/R-REC-M.1371/en
 """
 
+
```
Comment (schwehr) [inline ais/vdm.py]: 1 blank line

**Comment by geowurster** [comment]
@schwehr Fixed.  If you're happy with whats in the `NUMERIC_FIELDS` variables then this is should be ready to go.

Any idea if there's a Google Python Style configfile for [`pep8`](https://pypi.python.org/pypi/pep8), [`autopep8`](https://pypi.python.org/pypi/autopep8/), [`flake8`](https://pypi.python.org/pypi/flake8), [`yapf`](https://github.com/google/yapf) or some similar linter so contributors can run it before opening a PR?

**Comment by schwehr** [comment]
I'm asking if there is anything public for style guide checking.

**Comment by schwehr** [comment]
There isn't currently a config available for Google Style in the public form. :(

**Comment by geowurster** [comment]
Bummer.  I don't see the right flags in `pep8` or `flake8` either.

**Pull request merged**